.mailmap

```diff
@@ -0,0 +1,2 @@
+Jörn Zaefferer <joern.zaefferer@gmail.com>
+Richard D. Worth <rdworth@gmail.com>
```

Comment:
@rxaviers i am sorry i didn't get you. Do you mean there should be a new line at the end (after "Richard D. Worth") ?